core/src/main/java/novoda/lib/sqliteprovider/analyzer/Column.java

```diff
@@ -0,0 +1,37 @@
+package novoda.lib.sqliteprovider.analyzer;
+
+import novoda.lib.sqliteprovider.sqlite.IDatabaseMetaInfo;
+
+public class Column {
+    private final String name;
+    private final IDatabaseMetaInfo.SQLiteType type;
+
+    public Column(String name, IDatabaseMetaInfo.SQLiteType type) {
+        this.name = name;
+        this.type = type;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public IDatabaseMetaInfo.SQLiteType getType() {
+        return type;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Column column = (Column) o;
+
+        return name.equals(column.name);
```

Comment:
:+1: This should have `type` considered. I think I had a reason for this at the time but I can't think what it was now, and even then I should have added a comment explaining the reason.